.gitignore

```diff
@@ -61,9 +61,6 @@ npm-debug.log*
 .ci/bash_standard_lib.sh
 .gradle
 
-# apm plugin
-/x-pack/plugins/apm/tsconfig.json
```

Comment:
I had a look, and completionInfo() is still twice as fast with the optimized tsconfig, and project start up time also seems to be slower. I'm not sure if I'm doing something wrong. What do you think about leaving this in for now and removing it later if we feel like it's no longer worth it?

Reply:
I think we would be better served by concentrating any effort on further performance kibana-wide, rather than having a different setup for APM. I'm ok with leaving it in, but don't think we should.

Reply:
I agree in not spending any effort to improve this. So if it's causing issues let's remove it. But it worked well before, and I would like us to keep it around for a bit longer to compare the two different setups.